src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -34,26 +34,29 @@
     /**
      * A panel of normals can be a useful (optional) input to help filter out commonly seen sequencing noise that may appear as low allele-fraction somatic variants.
      */
-    @Argument(fullName="normal_panel", shortName = "PON", doc="VCF file of sites observed in normal", optional = true)
+    @Argument(fullName="normal_panel", shortName = "PON", doc="VCF file of sites observed in normal.", optional = true)
     public FeatureInput<VariantContext> pon;
 
     /**
      * A resource, such as gnomAD, containing population allele frequencies of common and rare variants.
      */
-    @Argument(fullName="germline_resource", doc="Population vcf of germline sequencing containing allele fractions", optional = true)
+    @Argument(fullName="germline_resource", doc="Population vcf of germline sequencing containing allele fractions.", optional = true)
     public FeatureInput<VariantContext> germlineResource;
 
     /**
      * Population allele fraction assigned to alleles not found in germline resource.
      */
-    @Argument(fullName="af_of_alleles_not_in_resurce", shortName = "default_af",
+    @Argument(fullName="af_of_alleles_not_in_resource", shortName = "default_af",
             doc="Population allele fraction assigned to alleles not found in germline resource.  A reasonable value is" +
                     "1/(2* number of samples in resource) if a germline resource is available; otherwise an average " +
                     "heterozygosity rate such as 0.001 is reasonable.", optional = true)
     public double afOfAllelesNotInGermlineResource = 0.001;
 
     /**
-     * Prior probability that any given site has a somatic allele
+     * Prior probability that any given site has a somatic allele. Impacts germline probability calculation.
+     * For example, -6 translates to one in a million or ~3000 somatic mutations per human genome.
+     * Depending on tumor type, mutation rate ranges vary (Lawrence et al. Nature 2013), and so adjust parameter accordingly.
+     * For higher expected rate of mutation, adjust number up, e.g. -5. For lower expected rate of mutation, adjust number down, e.g. -7.
```

Comment:
Added two sentences "The workflow uses this parameter only towards the germline event filter. It does NOT relate to the LOD threshold."